Tests/VSMTests/StatePublishingTests/ViewStatePublishingTests.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  ViewStatePublishingTests.swift
+//  
+//
+//  Created by Albert Bori on 2/28/23.
+//
+
+@testable import VSM
+import XCTest
+
+@available(iOS 14.0, *)
+final class ViewStatePublishingTests: StatePublishingTests {
+    
+    override func setUp() {
+        let viewState = ViewState(wrappedValue: MockState.foo)
```

Comment:
FYI: These tests spew out runtime warnings:
> Accessing StateObject's object without being installed on a View. This will create a new instance each time.

As far as I can tell, this has no bearing on the test outcomes and is a benign warning in a unit test scenario. It can be argued that SwiftUI types should not be part of unit tests and the underlying state container should be injected here, but anything else would be a test of the implementation instead of the outcome, to the detriment of covering all possible outcomes. No view-specific behavior is tested.